lcc_lokavaluto_app_connection/models/company.py

```diff
@@ -1,4 +1,4 @@
-from odoo import models, fields
+from odoo import models, fields, api
```

Comment:
not clear why this import is needed in this commit

Reply:
removed